backend/src/main/resources/application.yaml

```diff
@@ -108,6 +108,8 @@ simple-report:
       - support@simplereport.gov
     waitlist-recipient:
       - support@simplereport.gov
+    outreach-mailing-list-recipient:
```

Comment:
@fzhao99 @mpbrown Changed to Jayna's cdc email as per this conversation:
https://skylight-hq.slack.com/archives/C065V4DHBQF/p1710951757493149?thread_ts=1710865824.510909&cid=C065V4DHBQF

cc @alismx